rotkehlchen/chain/ethereum/modules/the_graph/constants.py

```diff
@@ -0,0 +1,9 @@
+from rotkehlchen.chain.evm.decoding.types import CounterpartyDetails
+
+CPT_THE_GRAPH = 'the_graph'
+
+THE_GRAPH_CPT_DETAILS = CounterpartyDetails(
+    identifier=CPT_THE_GRAPH,
+    label='The Graph',
+    image='',  # TODO: add image
```

Comment:
how to add this?

Reply:
we explained this in discord. Ask again if it does not make sense